core/src/scala/com/mygdx/game/screen/TiledScreen.scala

```diff
@@ -0,0 +1,51 @@
+package com.mygdx.game.screen;
+
+import com.badlogic.gdx.maps.tiled._;
+import com.badlogic.gdx.{ApplicationAdapter, Gdx, Input, InputProcessor, ScreenAdapter};
+import com.badlogic.gdx.graphics.{GL20, OrthographicCamera, Texture};
+import com.badlogic.gdx.graphics.g2d._;
+import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
+
+import com.mygdx.game.GameApplication
+
+class TiledScreen(game: GameApplication) extends ScreenAdapter{
+
+    var map : TiledMap = _;
```

Comment:
Usually in scala the `= _` is for delayed initialization aka when you _just can't_ initialize the variable at the time of construction. But here when you do actually initialize it, you're still in the constructor (Line 31)

Since you never have to change `map` why not:
```scala
val level = "..."
val map = new TmxMapLoader.load(level)
val renderer = new OrthogonalTiledMapRenderer(map, 1f/32f)
```